src/Common/src/Interop/Windows/mincore/Interop.Idna.cs

```diff
@@ -19,8 +19,7 @@ internal partial class mincore
                                         char* lpUnicodeCharStr,
                                         int cchUnicodeChar,
                                         [System.Runtime.InteropServices.OutAttribute()]
-
-                                        char* lpASCIICharStr,
+                                        IntPtr lpASCIICharStr,
```

Comment:
It's strange to have a P/Invoke like this that's marked unsafe and then has two pointer arguments, one of which is a ```char*``` and one of which is an ```IntPtr```.  This change is necessary due to an MCG bug?  Is there an issue tracking fixing MCG?  It'd be good to include a comment here indicating why this isn't a ```char*``` like the other, in part so that this doesn't get broken again.  That'll also help flag this as something we want to change back once the bug is fixed.  And if we decide instead to leave it like this, seems like it'd be better for both ```char*``` args to be changed to IntPtr for consistency, and unsafe removed from the signature.